src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Components/ComponentDesignTimeNodeWriter.cs

```diff
@@ -74,10 +74,25 @@ public override void WriteUsingDirective(CodeRenderingContext context, UsingDire
 
         if (node.Source is { FilePath: not null } sourceSpan)
         {
+            // Write in the form of
+            // using
```

Comment:
I wonder how `using static` and `using Alias = ...` are handled in Razor; do they also go through this code path?

Reply:
From memory, anything after the using keyword is simply passed through so they "work", but they don't really work as complete as a user would expect: https://github.com/dotnet/razor/issues/7670